src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const isActiveTabPresent = tabs.some(tab => tab.id === selectedTabId);
```

Comment:
Just use `find` method instead to find the selected tab

Reply:
Done